### ISSUE TITLE
Teleport Cooldown does nothing

### ISSUE THREAD

**Issue by Skyfay** [John-Paul-R/Essential-Commands#310]
Hey,
Im using the mod with Fabric 1.21.1.
I can set the teleport_cooldown option but it does nothing (also without OP). 

Any idea why?

Thanks :)

**Comment by John-Paul-R** [comment]
Yes, that option isn't implemented. I should probably just remove it for now. If I do ever add the ability to support cooldowns, it'll be via in command limits (https://github.com/John-Paul-R/command-limits/issues/1), as mentioned in https://github.com/John-Paul-R/Essential-Commands/issues/148. Which I'm interested in doing, but is currently not planned, due to my own irl priorities.

TL;DR currently, that option doesn't do anything, unfortunately.

**Comment by Skyfay** [comment]
Too bad, that would have been an important function for me. Thanks anyway.

**Comment by John-Paul-R** [comment]
tracking in #148
